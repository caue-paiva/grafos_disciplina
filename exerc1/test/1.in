10
4
2
1 2
5
4
0

Este caso testa a criacao de um grafo, a adicao de uma aresta e a impressao do grafo antes e depois da adicao da aresta.