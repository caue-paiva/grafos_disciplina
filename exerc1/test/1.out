Digite o numero de vertices inicial do grafo: 
OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: 
     1  2  3  4  5  6  7  8  9  10  
     ------------------------------
1: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
2: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
3: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
4: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
5: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
6: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
7: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
8: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
9: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
10: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
Numero de vertices: 10 , Numero de arestas bi-direcionais: 0 

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: Digite os numeros dos vertices que deseja adicionar uma aresta: Digite o peso da aresta: 
OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: 
     1  2  3  4  5  6  7  8  9  10  
     ------------------------------
1: [ -1  5 -1 -1 -1 -1 -1 -1 -1 -1 ]
2: [  5 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
3: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
4: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
5: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
6: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
7: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
8: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
9: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
10: [ -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 ]
Numero de vertices: 10 , Numero de arestas bi-direcionais: 1 

OPCOES
1 - Adicionar vertice
2 - Adicionar aresta
3 - Remover aresta
4 - Mostrar vertices e arestas
5 - Mostrar lista de vertices adjacentes
6 - Remover aresta de menor weight
7 - Mostrar numero de vertices
8 - Verificar existencia de aresta
0 - Sair
Digite sua opcao: